src/binder/bindertracing.cpp

```diff
@@ -68,6 +77,82 @@ namespace
             &activityId);
 #endif // FEATURE_EVENT_TRACE
     }
+
+    void GetAssemblyLoadContextNameFromSpec(AssemblySpec *spec, /*out*/ SString &alcName)
+    {
+        _ASSERTE(spec != nullptr);
+
+        AppDomain *domain = spec->GetAppDomain();
+        ICLRPrivBinder* bindContext = spec->GetBindingContext();
+        if (bindContext == nullptr)
+            bindContext = spec->GetBindingContextFromParentAssembly(domain);
+        
+        _ASSERTE(bindContext != nullptr);
+
+        UINT_PTR binderID = 0;
+        HRESULT hr = bindContext->GetBinderID(&binderID);
+        _ASSERTE(SUCCEEDED(hr));
+        if (FAILED(hr))
+            return;
+
+        ICLRPrivBinder *binder = reinterpret_cast<ICLRPrivBinder *>(binderID);
+        if (AreSameBinderInstance(binder, domain->GetTPABinderContext()))
+        {
+            alcName.Set(W("Default"));
+        }
+#ifdef FEATURE_COMINTEROP
+        else if (AreSameBinderInstance(binder, domain->GetWinRtBinder()))
```

Comment:
I would display the WinRT binder in tracing as "Default" context as well. The separate binder for WinRT is internal implementation detail that would be best to hide.